### PR TITLE
Replace MSDN links in Visual Basic area

### PR DESCRIPTION
**Pull request by nemrism**
## Summary

- applied link replacement pushed to the C# area (#7484)
- replaced other links pointing to the VB area
- standardize `See also` section on file changed (#6923)

Contributes to #2037

### CODE REVIEW

File: docs/visual-basic/developing-apps/windows-forms/index.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ An important part of Visual Basic is the ability to create Windows Forms applica
 |To|See|  
 |--------|---------|  
 |Use the <xref:System.Windows.Forms.BindingSource> component|[How to: Bind Windows Forms Controls with the BindingSource Component Using the Designer](../../../framework/winforms/controls/bind-wf-controls-with-the-bindingsource.md)|  
-|Work with [!INCLUDE[vstecado](~/includes/vstecado-md.md)] data sources|[How to: Sort and Filter ADO.NET Data with the Windows Forms BindingSource Component](https://msdn.microsoft.com/library/ya3sah92.aspx)|  
+|Work with [!INCLUDE[vstecado](~/includes/vstecado-md.md)] data sources|[How to: Sort and Filter ADO.NET Data with the Windows Forms BindingSource Component](../../../framework/winforms/controls/sort-and-filter-ado-net-data-with-wf-bindingsource-component)|
```
Comment (rpetrusha) [inline docs/visual-basic/developing-apps/windows-forms/index.md]: the link is broken; there should be an ".md" file extension.

File: docs/visual-basic/misc/bc30817.md [hunk 2]
```diff
@@ -17,7 +17,8 @@ It is no longer valid to use the value of a variable or expression with the `GoT
...
   
-## See Also  
- [On Error Statement](../../visual-basic/language-reference/statements/on-error-statement.md)  
- [If...Then...Else Statement](../../visual-basic/language-reference/statements/if-then-else-statement.md)  
- [Case (Visual Basic)](https://msdn.microsoft.com/library/a14efce6-5057-4b7d-8afd-056dd4abdcee)
+## See also
+
+- [On Error Statement](../../visual-basic/language-reference/statements/on-error-statement.md)  
+- [If...Then...Else Statement](../../visual-basic/language-reference/statements/if-then-else-statement.md)  
+- [Select...Case Statement](../../visual-basic/language-reference/statements/select-case-statement)
```
Comment (rpetrusha) [inline docs/visual-basic/misc/bc30817.md]: the link is broken; it should include the file extension (".md")

File: docs/visual-basic/misc/bc32023.md [hunk 3]
```diff
@@ -17,7 +17,8 @@ The group variable specified in a `For Each` statement is not a collection objec
...
   
-## See Also  
- <xref:System.Collections.IEnumerable>  
- [For Each...Next Statement](../../visual-basic/language-reference/statements/for-each-next-statement.md)  
- [Visual Basic Collection Class](https://msdn.microsoft.com/library/0cb2d1ad-c58d-42c0-8e69-d81f5a15e532)
+## See also
+
+- <xref:System.Collections.IEnumerable>  
+- [For Each...Next Statement](../../visual-basic/language-reference/statements/for-each-next-statement.md)  
+- [Visual Basic Collection Class](../../visual-basic/programming-guide/concepts/collections.md#visual-basic-collection-class.md)
```
Comment (rpetrusha) [inline docs/visual-basic/misc/bc32023.md]: The bookmark does not exist; the build system only generates bookmarks automatically for level-2 heads, not level-3 heads. Try the bookmark of the parent section: #kinds-of-collections

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks, @nemrism, for continuing to add bullets to the see also items as well as for replacing MSDN links. There are three build issues that need to be fixed, then I'll merge.

File: docs/visual-basic/misc/bc32023.md (hunk 3 above)
Comment (nemrism) [reply docs/visual-basic/misc/bc32023.md]: I took the bookmark directly from the website page:
https://docs.microsoft.com/en-us/dotnet/visual-basic/programming-guide/concepts/collections#visual-basic-collection-class

But I see that I've added a wrong '.md' ext at the end of the bookmark

**Comment by nemrism** [comment]
Let me know if it works for you like this @rpetrusha

**Comment by rpetrusha** [comment]
Yes, it works for me. Thanks for making the corrections, @nemrism. There was one additional typo that I've just fixed. When the build finishes, if it looks good, I'll merge your PR.

**Review by rpetrusha — APPROVED** [review]

**Pull request merged**